streaming/src/main/scala/org/apache/spark/streaming/dstream/DStreamCheckpointData.scala

```diff
@@ -102,8 +102,8 @@ class DStreamCheckpointData[T: ClassTag] (dstream: DStream[T])
   }
 
   /**
-   * Restore the checkpoint data. This gets called once when the DStream graph
-   * (along with its DStreams) are being restored from a graph checkpoint file.
+   * Restore the checkpoint data. This gets called once while the DStream graph
```

Comment:
I'd leave "when" in this case since it is _called_ at the time the graph is restored, not sometime during that process